test/JDBC/input/BABEL-3731.sql

```diff
@@ -0,0 +1,39 @@
+SELECT (case when count(*)>=0 THEN 1 END) FROM dbo.sysprocesses;
+GO
+
+-- should ignore white space between schema name and view name
+SELECT (case when count(*)>=0 THEN 1 END) FROM dbo. sysconfigures;
+GO
+
+SELECT (case when count(*)>=0 THEN 1 END) FROM dbo.             syscurconfigs;
+GO
+
+-- case-insensitive
+SELECT (case when count(*)>=0 THEN 1 END) FROM DBo.SySObjEcTS
```

Comment:
`sys.sysobjects` is database specific. Could you add tests to verify that after rewriting the schema name, the results are correct?

Create an object (maybe table1) in db1. Go to master and check using the cross-db query "db1.dbo.sysobjects" that it's showing table1.

Reply:
Added the tests in the next-revision in the respective JDBC test file.